### ISSUE TITLE
plot_approval function in anes_functions.R throws an error

### ISSUE THREAD

**Issue by sjgenco** [czep/HTAD#2]
Working my way thru your excellent ANES EDA article, I ran into another problem, hoping it's easy to fix.

I ran this command: `t1 %>% plot_approval(V201144x, "President handling COVID-19")`
and got this response:
`Error: Must group by variables found in '.data. * Column 'V201144x is not found. Run 'rlang::last_error()' to see where the error occurred.`
Same error occurs with all the 'approval' plots.
Here's the traceback:
```
13. | stop(fallback)
12. | signal_abort(cnd)
11. | abort(c("Must group by variables found in `.data`.", glue("Column `{unknown}` is not found.")))
10. | group_by_prepare(.data, ..., .add = .add, caller_env = caller_env())
9. | group_by.data.frame(., preintent, { { varname } ...
8. | group_by(., preintent, { { varname } ...
7. | summarize(., n = sum(V200010a))
6. | filter(., !is.na({ { varname } ...
5. | mutate(., p = n/sum(n), v = to_factor({ { varname } ...
4. | ggplot(., aes(x = v, y = p, group = preintent, fill = preintent))
3. | data %&gt;% filter(!is.na(preintent)) %&gt;% group_by(preintent, { { varname } ... at&nbsp;anes_functions.R#120
2. | plot_approval(., V201144x, "President handling COVID-19")
1. | t1 %&gt;% plot_approval(V201144x, "President handling COVID-19")
```
Everything else has worked perfectly up to this point. Thanks again for all your work on this.

**Comment by czep** [comment]
There's a code chunk just above the calls to `plot_approval` in lines 460-470 which sets t1 as a data frame containing the missing values applied for the approval variables.  Try running that statement before running the plots.  I think that will make sure that `t1` is properly setup for the approval plots.

**Comment by sjgenco** [comment]
thank you! I was interested in the COVID approval numbers, so I jumped there and didn't see that block of na range definitions. As a rather meager thank you gift, I liked the way your ci ranges were formatted in your article, but noticed they were not formatted that way when I ran your code in RStudio. So I deployed my minimal html/css formatting skills and added a style designation to your `reportci` function:
```
reportci <- function(p, lo, hi) {
  pctp <- percent(p, accuracy = 0.02)
  pctlo <- percent(lo, accuracy = 0.02)
  pcthi <- percent(hi, accuracy = 0.02)
  paste0(
    '<span class="pctp">', pctp, '</span>',
    '<br />',
    '<span class="pctci", style="font-size:0.8em;color:#808080">(', pctlo,
    ', ',
    pcthi,
    ')</span>'
  )
}
```
Not a groundbreaking addition, but the tables look better now.
![t1](https://user-images.githubusercontent.com/19754884/120087320-a96eb200-c09b-11eb-8149-5fb1b32fc55a.jpg)
